src/component/tooltip/TooltipView.ts

```diff
@@ -365,11 +370,13 @@ class TooltipView extends ComponentView {
         else if (payload.x != null && payload.y != null) {
             // FIXME
             // should wrap dispatchAction like `axisPointer/globalListener` ?
-            api.dispatchAction({
-                type: 'updateAxisPointer',
-                x: payload.x,
-                y: payload.y
-            });
+            if (tooltipModel.option.trigger === 'axis') {
```

Comment:
Can you explain why this action should only be dispatched for axis triggers?

Reply:
I found this action causes the `emphasis` state lost.
After dispatch:
<img width="201" alt="截屏2025-03-11 17 10 48" src="https://github.com/user-attachments/assets/a2dc5a1c-4164-4a69-94a1-2fdb41837daa" />
Expected:
<img width="189" alt="截屏2025-03-11 17 17 49" src="https://github.com/user-attachments/assets/1f809bdf-036a-451c-8d16-d81e3a04952e" />
Maybe dispatch this action only when `axisPointer` is enabled would be better.
Is there any way to check if `axisPointer` is enabled or not?

Reply:
Logically speaking, I don't think whether the `axisPointer` is enabled makes any difference in this regard. If we consider that `setOption` should lead to an update of the tooltip, then it is also logical that `setOption` causes the `emphasis` state to become updated.

This is also one of the reasons why I'm more inclined to think that the issue in this PR isn't a bug. Perhaps keeping the setting of "changing the 'trigger' via setOption won't trigger the redrawing of the tooltip" would simplify matters. May I ask what initially led you to need to fix the problem in this PR?

Reply:
I will revert this change because you think losing the "emphasis" is not a bug.
But changing the tooltip "trigger" without updating the tooltip is certainly a bug.

Reply:
done